static/js/add-shows/new-show.js

```diff
@@ -41,8 +41,10 @@ MEDUSA.addShows.newShow = function() {
         sampleText += '</b>';
 
         // if we have a show name then sanitize and use it for the dir name
-        if (showName.length) {
-            $.get('addShows/sanitizeFileName', {name: showName}, function (data) {
+        if (showName.length > 0) {
```

Comment:
@OmgImAlexis why checking > 0 here and not "!=" too?   can't we always use ".length > 0" instead of != or !== ?